code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -74,6 +74,8 @@
 	id = "capsaicincondensation"
 	results = list("condensedcapsaicin" = 5)
 	required_reagents = list("capsaicin" = 1, "ethanol" = 5)
+	
+////////////////////////////////// Lye recipes ///////////////////////////////////
```

Comment:
You do know lye is the "erase all life" bit of soap and not the "water soluble" bit of soap right?